### ISSUE TITLE
FE - Prefill Pattern Task Updates - Post Pilot Fixes

### ISSUE THREAD

**Issue by adamwhitlock1** [department-of-veterans-affairs/tmf-auth-exp-design-patterns#86]
### User Story
As an engineer I would like to update a few small bugs and issues with the current state out our current pilot task prototypes.

General

- make the sign in button in the header do the mock auth action (if possible)
- for dob use the abbreviated wording on mobile and the fully spelled out month on desktop. See this convo from slack: https://dsva.slack.com/archives/C07909N7U8Z/p1723818181190559

Green Task

- Editing of the mailing address should have a 'nametag' component with matching name of mock user. Right now there is dummy data that does not match the mock user that is loaded.

Yellow Task

- Add ')' to the tty section of the yellow task in the info section of the first vet info page (ssn dob and name page)
- Update the page title for Task Yellow to be what the actual form name is 'Update your VA health benefits information' with the subtitle: 'Health Benefits Update Form (VA Form 10-10EZR)'
- Add success message to yellow task after save mailing address success

Purple Task

- Get the updated phone number to show on the purple task after save success
- On the edit page, make sure the home phone number field is blank and not prefilled with the existing number.
- Change Note in purple task about updates only reflecting on form and not that it will update the profile data
- Update phone save success message to not save 'phome'

### Background

After pilot we found these issues, so we want to update our code to fix them.

### ACs
- [x] each item is addressed
- [x] codespace is reflective of these changes
- [ ] changes reviewed with team

**Comment by adamwhitlock1** [comment]
We can split this up if we want, but thought we should get all these items in a hit list to knock out.

**Comment by msbtterswrth** [comment]
@adamwhitlock1 i've added this to the sprint, could you point this and lmk when  you think it might be done?

**Comment by adamwhitlock1** [comment]
We have pointed it for an estimate of 3 since we have several things to tackle and a few things have been tripping us up with the logic around the save logic. We should be done with this by the end of the sprint. Adam is working on the navigation page #84 next, but will be focusing on updates around this ticket for green/yellow once that is done, and Belle has already started working on the purple updates.

**Comment by adamwhitlock1** [comment]
Updates have been addressed in Belle's PR: https://github.com/department-of-veterans-affairs/vets-website/pull/31430 for the purple task.

Updates for the green and yellow tasks are made in this PR from Adam: https://github.com/department-of-veterans-affairs/vets-website/pull/31483 along with the navigation page. Once these PRs are approved and merged, then we will review the latest changes with the team.

**Comment by adamwhitlock1** [comment]
We have merged these fixes and deployed a new codespace.

I had to change the url of the codespace, because the old codespace would throttle down after 30 minutes of inactivity, even though I had updated the settings to be 4 hours instead.
The reason for this is that the codespace was created before I updated that setting, and so I had to destroy it / make a new one with a new url for the new setting to take. From here on out that should no longer be an issue.

https://musical-doodle-7676579pxr3rxrg-3001.app.github.dev/mock-form-ae-design-patterns


This task is ready to be reviewed and validated / closed as time permits.

**Comment by msbtterswrth** [comment]
Everything looked great to me on our qa call yesterday! I'll close this and we can open a new task if we find anything else, great job team!
